docs/source/concepts/time.rst

```diff
@@ -0,0 +1,90 @@
+.. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _time:
+
+Time
+####
+
+The DAML language contains a function :ref:`getTime <daml-ref-gettime>` which returns the “current time”.
+However, the notion of time comes with a lot of problems in a distributed setting.
+
+This document describes the detailed semantics of time on DAML ledgers,
+centered around the two timestamps assigned to each transaction:
```

Comment:
Some thinking out loud: I'm wondering what the best way to link this to the ledger model is. The model has a notion of a commit, and the `lt/rt` would in reality be assigned to commits. I don't think it's urgent to change/solve it for this PR, as I think that just saying "transaction" will be sufficiently understandable.

Changing lt/rt to refer to commits also introduces problems, though, since the ledger model also has a notion of projections, which are just transactions, but we still want to assign these timestamps to them. I see a couple of ways out:
1. we assign the timestamps to both a commit and to every action in the commit, with all actions within a single commit having the same timestamp
2. we change the notion of a projection to be a commit, though this one is tricky, as we have to figure out what to do with requesters, such that we don't mess the authorization definitions up (likely: a commit would then have to allow for a different requester for each top-level action in the commit).

Reply:
I'd pick option 1:

Define that the ledger time is a property of the commit, then define that everything that "belongs" to a commit (all actions, all contracts created by that commit) have a ledger time equal to the ledger time of the commit.

The current text already has the sentence "where lt_C is the ledger time of the transaction that created C".

Reply:
@oggy- is this something you would like to resolve now?